Formula/zsh-git-prompt.rb

```diff
@@ -0,0 +1,28 @@
+class ZshGitPrompt < Formula
+  desc "Informative git prompt for zsh"
+  homepage "https://github.com/olivierverdier/zsh-git-prompt"
+  url "https://github.com/olivierverdier/zsh-git-prompt/archive/v0.5.tar.gz"
+  sha256 "87e5a908369f402e975426ffd61a8800f1c04c0a293f1d4015a6fb1f4408e77d"
+
+  bottle :unneeded
+
+  def install
+    inreplace "README.md", "path/to", opt_prefix
+    prefix.install Dir["*.{sh,py}"]
+  end
+
+  def caveats; <<-EOS.undent
+    First, make sure zsh-git-prompt is loaded from your zshrc:
```

Comment:
Use "your .zshrc" to be consistent with caveats of other zsh-related stuff. Other than that I'm happy with the caveats.